resqpy/surface.py

```diff
@@ -1534,6 +1535,14 @@ def convert_to_irap(self, file_name):
          for item in lines:
             f.write(item)
 
+   def append_extra_metadata(self, meta_dict):
```

Comment:
Would this make sense as a general method of the base class `BaseResqpy`, or is it PointSet specific? Also, perhaps the word "extra" in the title is redundant?